packages/cuba-react-ui/src/ui/paging/Paging.tsx

```diff
@@ -0,0 +1,123 @@
+import {PaginationConfig} from "antd/lib/table";
+import {DataCollectionStore} from "@cuba-platform/react-core";
+import * as QueryString from "querystring";
+import React from "react";
+import {Pagination} from "antd";
+import {toJS} from "mobx";
+
+type Props = {
+  paginationConfig: PaginationConfig;
+  total?: number,
+  onPagingChange: (current: number, pageSize?: number) => void;
+}
+
+/**
+ * Paging defaults
+ */
+export const defaultPagingConfig: PaginationConfig = {
+  current: 1,
+  pageSize: 10,
+  defaultPageSize: 10,
+  showSizeChanger: true,
+  pageSizeOptions: ['10', '20', '30']
```

Comment:
Let's stick with page sizes like in generic UI 20,50,100,500,1000,5000

Reply:
Done.